src/Console/ServeCommand.php

```diff
@@ -0,0 +1,40 @@
+<?php namespace TightenCo\Jigsaw\Console;
+
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class ServeCommand extends Command
+{
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    protected function configure()
+    {
+        $this->setName('serve')
+            ->setDescription('Serve local site with php built-in server.')
+            ->addArgument(
```

Comment:
Thanks for the feedback Pete and the detailed explanation. 
I did note in the PR description that i used arguments to match what Adam had done in InitCommand. Was trying to match the style the project was already using. 

Happy to make this change. Perhaps then InitCommand should be updated to use options as well, so all commands are consistent?